src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -727,6 +723,7 @@ static void create_rr_locked(grpc_exec_ctx *exec_ctx, glb_lb_policy *glb_policy,
   /* Connectivity state is a function of the RR policy updated/created */
   update_lb_connectivity_status_locked(exec_ctx, glb_policy, rr_state,
                                        rr_state_error);
+  GRPC_ERROR_UNREF(rr_state_error);
```

Comment:
This looks wrong.  I think `update_lb_connectivity_status_locked()` should take ownership of the error, as per:

https://github.com/grpc/grpc/blob/master/doc/core/grpc-error.md#rule-3